starter-code/scripts/views/articleView.js

```diff
@@ -35,6 +41,17 @@
   };
 
   // COMMENT: What does this method do?  What is it's execution path?
+
+  /*  handleFilters is invoked by articleView.index.  It takes no arguments.  It selects the unordered list with the id "filters" and invokes the .one method on it.  .one attaches an event handler to an event for the elements.  The handler is execture at most once per element per event type.  In this case, 'change' and 'select' are the events.  It invokes as a handler an anonymous callback function.
```

Comment:
Is line 56 attaching two events, "change" and "select"? Since jQuery 1.7, what is the second parameter of a .one method? It functions very similarly to .on, so what was the second parameter when we were setting up delegated event handlers?

You have the functionality correct in the rest of this comment section, but the explanation of what the handler is being attached to isn't accurate.